test/parallel/test-stream-pipe-await-drain.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 const common = require('../common');
 const stream = require('stream');
+const assert = require('assert');
 
 // This is very similar to test-stream-pipe-cleanup-pause.js.
 
 const reader = new stream.Readable();
 const writer1 = new stream.Writable();
 const writer2 = new stream.Writable();
+const writer3 = new stream.Writable();
```

Comment:
I thought it's better to trace if the counter is not stuck on value 1 if we have multiple writers in pipeline. For example, if someone will crash the behavior of this counter as a flag he will get error in this test.